Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/LCPConstraintSolver.h

```diff
@@ -107,8 +107,8 @@ class SOFA_COMPONENT_CONSTRAINT_LAGRANGIAN_SOLVER_API LCPConstraintSolver : publ
     void removeConstraintCorrection(core::behavior::BaseConstraintCorrection *s) override;
 
 private:
-    std::vector<core::behavior::BaseConstraintCorrection*> constraintCorrections;
-	std::vector<char> constraintCorrectionIsActive; // for each constraint correction, a boolean that is false if the parent node is sleeping
+    type::vector<core::behavior::BaseConstraintCorrection*> constraintCorrections;
+    type::vector<bool> constraintCorrectionIsActive; // for each constraint correction, a boolean that is false if the parent node is sleeping
```

Comment:
A good change, but this set this PR as "officially" breaking 🤫

Reply:
you're right